uui/components/datePickers/__tests__/__snapshots__/DatePickerBody.test.tsx.snap

```diff
@@ -2,612 +2,608 @@
 
 exports[`DataPicker should be rendered correctly 1`] = `
 <div
-  className="uui-datepicker-container"
```

Comment:
The only difference here is removed wrapping `uui-datepicker-container` previously.

Reply:
See that `props.cx` were applied here https://github.com/epam/UUI/blob/186fd0ee378a7f1aeef8b2c1f4f3ca0890d7bc67/uui-components/src/inputs/DatePicker/DatePickerBodyBase.tsx#L52

The same `props.cx` were applied here. So it looks like duplication.
https://github.com/epam/UUI/blob/186fd0ee378a7f1aeef8b2c1f4f3ca0890d7bc67/uui/components/datePickers/DatePickerBody.tsx#L69

@AlekseyManetov Do we still need to apply the following to `DatePickerBody` and were to apply `props.cx`? 
```
ref={ this.props.forwardedRef } 
className={ cx(uuiDatePickerBodyBase.container, this.props.cx) } 
{ ...this.props.rawProps }
```

Reply:
I think we need to preserve the previous structure with global CSS classes to avoid causing breaking changes, as some apps might rely on it.
The cx prop, along with rawProps, should be passed to the root component node.